### PR TITLE
Alternative readings, instead of MDM and SpeakinJS

### PR DESCRIPTION
**Pull request by gijscor**
Both resources are very far from beginner friendly and clearly suppose previous coding experiences. 
Example of a sentence used in these resources:

"javaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by supplementing it with additional objects"

As the level of English of some of our students is limited, and many have no background in programming, I think these resources are mostly just having students read information that they are not yet ready to process, and mostly will leave them confused. There are much better resources (that we have written ourselves) that explain it much better, and that we don't link to in the reading material of this week. (I have created a seperate pull request for this.

### CODE REVIEW

File: Week2/README.md [hunk 1]
```diff
@@ -25,6 +25,15 @@ Only watch the below chapters:
 2. Core Programming Syntax 
 3. Variables and Data Types
 
+Read up on the most important topics of next week:
+
+[Variables (var, let, const)](http://javascript.info/variables)
+[Basic Data types (Strings, Numbers, Arrays, Booleans)](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/values.md)
+[Operators](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/operators.md)
+[Special characters and their names](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/names_of_special_characters.md)
+[Naming conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md)
```
Comment (remarcmij) [inline Week2/README.md]: Hi @gijscor, this doesn't show up as a list when rendered as HTML. Check the link if your Markdown is a bit rusty.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists

File: Week2/README.md [hunk 2]
```diff
@@ -25,6 +25,15 @@ Only watch the below chapters:
...
+
+
 - Read this ~ http://speakingjs.com/es5/ch01.html read up to and including the *Strings* chapter (it’s okay if you don’t understand all of it yet, we will cover these concepts in class as well. Do make sure to write or document the questions you have so we can discuss them in class)
 
 - Read the entire JavaScript Introduction at MDN~ https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Introduction
```
Comment (remarcmij) [inline Week2/README.md]: I think your other PR (which I already merged) was to get rid of these two readings.

**Review by remarcmij — CHANGES_REQUESTED** [review]
See comments.

File: Week2/README.md [hunk 3]
```diff
@@ -26,22 +26,22 @@ Only watch the below chapters:
...
 
-- Intro JavaScript (What is it, where can you use it for)  
-- [Variables (var, let, const)](./../../../../fundamentals/blob/master/fundamentals/variables.md)  
-- [Basic Data types (Strings, Numbers, Arrays, Booleans)](./../../../../fundamentals/blob/master/fundamentals/values.md)  
-- [Operators](./../../../../fundamentals/blob/master/fundamentals/operators.md)  
-- [Special characters and their names](./../../../../fundamentals/blob/master/fundamentals/names_of_special_characters.md)  
-- [Naming conventions](./../../../../fundamentals/blob/master/fundamentals/naming_conventions.md)
+- Helpful resource: http://jsbooks.revolunet.com/ (here you can find tons of free JavaScript books online)
+
+- The 50 best websites to Learn JavaScript: http://www.codeconquest.com/blog/top-50-websites-to-learn-javascript/
```
Comment (M3kH) [inline Week2/README.md]: This is just confusing and doesn't provide any quality insight.

**Review by M3kH — CHANGES_REQUESTED** [review]
Even if the title of this PR says *Alternative Reading* this PR doesn't provide any additional value.

I would rather close it.

File: Week2/README.md (hunk 3 above)
Comment (gijscor) [reply Week2/README.md]: I agree, that was already in the original document though.

File: Week2/README.md (hunk 1 above)
Comment (gijscor) [reply Week2/README.md]: solved

File: Week2/README.md (hunk 2 above)
Comment (gijscor) [reply Week2/README.md]: removed them again

**Comment by gijscor** [comment]
@M3kH ready to merge now?

**Review by remarcmij — APPROVED** [review]

**Review by M3kH — APPROVED** [review]

**Pull request merged**